developer/src/kmc/README.md

```diff
@@ -1,105 +1,191 @@
-Keyman Developer - Next Generation Compiler
+Keyman Developer - Command Line Compiler
 ================
 
-This package provides the following Keyman **command line tools**:
+This package provides a command-line interface to the Keyman Developer compiler
+toolchain, `kmc`.
 
- - `kmc` — takes **LDML Keyboard .xml sources** and compiles them in to a
-   KMXPlus **.kmx** file.
- - `kmlmc` — takes **lexical model sources** and compiles them in to a **.js**
-   file.
- - `kmlmp` — uses a `.model.kmp` file to generate a redistributable **lexical
-   model package**.
+## Install kmc
 
-`kmlmc` is intended to be used standalone, or as part of a build system. `kmlmp`
-is used only by command line tools.
+`kmc` is available as:
+* a part of Keyman Developer (Windows)
```

Comment:
Just to disambiguate from Keyman for Windows

```suggestion
* a part of Keyman Developer (Windows only)
```

Plus is it matches your heading later (integration?)